.pre-commit-config.yaml

```diff
@@ -18,6 +18,6 @@ repos:
         args:
           - --fix=lf
   - repo: https://github.com/crate-ci/typos
-    rev: v1.17.2
+    rev: typos-v0.10.21
```

Comment:
This is a downgrade. `pre-commit autoupdate` doesn't handle the tags correctly in this repo.

Reply:
ok, good catch: I'm sorry, I didn't realize. Will look into it tmr.

Reply:
so, what do you do to get this right? Set them manually?